docker/runtime-entrypoint.sh

```diff
@@ -1,82 +1,95 @@
 #!/usr/bin/env bash
 
-# show the commands we are executing
+# Show the commands we are executing
 set -x
-# exit if a command fails
+# Exit if a command fails
 set -e
-# exit if any command in a pipeline fails
+# Exit if any command in a pipeline fails
 set -o pipefail
 
-echo "Config variables:"
-echo "NETWORK=$NETWORK"
-echo "RPC_PORT=$RPC_PORT"
-echo "LOG_FILE=$LOG_FILE"
-
-echo "Config location:"
-echo "ZEBRA_CONF_DIR=$ZEBRA_CONF_DIR"
-echo "ZEBRA_CONF_FILE=$ZEBRA_CONF_FILE"
-echo "ZEBRA_CONF_PATH=$ZEBRA_CONF_PATH"
+# Set this to change the default cached state directory
+# Path and name of the config file
+: "${ZEBRA_CONF_DIR:='/etc/zebrad'}"
+: "${ZEBRA_CONF_FILE:='zebrad.toml'}"
+if [[ -n "$ZEBRA_CONF_DIR" ]] && [[ -n "$ZEBRA_CONF_FILE" ]]; then
+    ZEBRA_CONF_PATH="$ZEBRA_CONF_DIR/$ZEBRA_CONF_FILE"
+fi
 
-echo "Other variables:"
-echo "SHORT_SHA=$SHORT_SHA"
-echo "SENTRY_DSN=$SENTRY_DSN"
+# [network]
+: "${NETWORK:='Mainnet'}"
+: "${ZEBRA_LISTEN_ADDR:='127.0.0.1'}"
```

Comment:
This changes the default from 0.0.0.0, which is a breaking change. The Zcash listener is expected to be available outside the container.

```suggestion
: "${ZEBRA_LISTEN_ADDR:='0.0.0.0'}"
```